modules/mono/glue/GodotSharp/GodotSharp/Core/NativeInterop/ExceptionUtils.cs

```diff
@@ -122,6 +153,8 @@ public static void LogUnhandledException(Exception e)
         {
             try
             {
+                InvokeUserUnhandledExceptionReporter(e);
```

Comment:
For the same reason, this line is not needed. Users can use `AppDomain.CurrentDomain.UnhandledException`

Reply:
do you mean that users can use 
`AppDomain.CurrentDomain.UnhandledException` as it stands now? because if i remember correctly, i think that event is no longer fired on unhandled exceptions, so binding to it does nothing

Reply:
> do you mean that users can use `AppDomain.CurrentDomain.UnhandledException` as it stands now? because if i remember correctly, i think that event is no longer fired on unhandled exceptions, so binding to it does nothing

This code will throw an exception that it will be caught by the `AppDomain.CurrentDomain.UnhandledException`.
```C#
new Thread(() => throw new Exception()).Start();
```

So, users only need to add its handlers to `AppDomain.CurrentDomain.UnhandledException` and to the new `Godot.Logging.UnhandledExceptionReporter` from this PR.

Reply:
In fact, to catch unhandled errors from code like this `new Task(() => throw new Exception()).Start();`, the `TaskScheduler.UnobservedTaskException` should send the error to the debugger, like `AppDomain.CurrentDomain.UnhandledException` is already doing in `OnCoreApiAssemblyLoaded`. What do you think @raulsntos ?